Assets/Plugins/Source/Core/EOSCreateOptions.cs

```diff
@@ -44,7 +44,7 @@ public class EOSCreateOptions : IEOSCreateOptions
 
         #region Platform-Specific properties
 
-#if UNITY_PS5 || UNITY_GAMECORE || UNITY_SWITCH || UNITY_ANDROID || UNITY_IOS
+#if !(UNITY_STANDALONE_WIN || UNITY_EDITOR_WIN || UNITY_STANDALONE_LINUX || UNITY_EDITOR_LINUX || UNITY_STANDALONE_OSX || UNITY_EDITOR_OSX)
         public IntegratedPlatformOptionsContainer IntegratedPlatformOptionsContainerHandle { get => options.IntegratedPlatformOptionsContainerHandle; set => options.IntegratedPlatformOptionsContainerHandle = value; }
 #endif
```

Comment:
This was changed so that the conditional for the existence of the property as defined on the _interface_ would be the same as the conditional for the _implementation_ of the property.